extra_model/_run.py

```diff
@@ -6,12 +6,14 @@
 from extra_model._models import ExtraModel
 
 MODELS_FOLDER = "./embeddings"
-OUTPUT_FILE = "result.csv"
+OUTPUT_FILE = Path("result.csv")
 
 logger = logging.getLogger(__name__)
 
 
-def run(input_path: Path, output_path: Path) -> None:
+def run(
```

Comment:
You also need to add this parameter to `_cli.py`.
It is also a good idea to add an example of using this parameter to README.

Reply:
Also, what happens if people don't put `.csv` in the end of the name? Do we want to enforce that?

Reply:
added the argument to `_cli.py`. On the second point, I would not enforce a file extension but curious to hear your opinions on this @romatik @chrisantonellis

Reply:
it's a bit of a footgun, but given that we add info into the help and an example to the README, it should be alright.